packages/vscode-extension/src/webviews/FeedbackFormWebview.ts

```diff
@@ -0,0 +1,196 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { VSTranslationService } from '../service/VSTranslationService';
+import { DEVASSIST_SERVICE } from '../service/TranslationKeys';
+import { getDevAssistCurrentSettings } from '../startup/DevAssistConfiguration';
+import { ApplicationConstants, FileUtils } from '../util/ExtensionUtil';
+import VSConsoleLogger from '../loggers/VSConsoleLogger';
+import {
+	validateIntegerWithinInterval,
+	validateMultipleOptionField,
+	validateTextAreaField,
+} from './WebviewFieldValidationUtils';
+import { DEVASSIST } from '../ApplicationConstants';
+
+const translationService = new VSTranslationService();
+const vsLogger = new VSConsoleLogger();
+
+const MEDIA_DIR = 'resources/media'
+const PROXY_URL = DEVASSIST.PROXY_URL;
+
+const WEBVIEW_FILE_NAMES = {
+	FEEDBACK_FORM : {
+		HTML : 'FeedbackForm.html',
+		CSS : 'FeedbackForm.css'
+	},
+	SUBMITTING_HTML : 'FeedbackFormSubmitting.html',
+	SUCCESS_HTML : 'FeedbackFormSuccess.html',
+	FAILURE_HTML : 'FeedbackFormFailure.html',
+}
+
+const WEBVIEW_EVENTS = {
+	CLOSE : "CLOSE_WEBVIEW",
+	SUBMIT_FEEDBACK : "SUBMIT_FEEDBACK",
+	OPEN_NEW_FEEDBACK_FORM : "OPEN_NEW_FEEDBACK_FORM",
+}
+
+type FeedbackFormData = {
+	feedback: string;
+	topics: string[];
+	rating: number;
+};
+
+const VALID_FEEDBACK_TOPICS = [
+	"CodeExplanation",
+	"SDFObjectGeneration",
+	"SuiteScriptCodeGeneration",
+	"UnitTesting",
+	"Other"
+]
+
+let feedbackFormPanel: vscode.WebviewPanel | undefined;
+let vscodeExtensionMediaPath : string;
```

Comment:
I would like to have a service that can get the html content instead of constructing the path each time. It will make the implementation cleaner.